.github/workflows/ci.yml

```
@@ -1,40 +1,29 @@
-name: Node CI
+name: CI
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
     strategy:
-      fail-fast: false
       matrix:
-        node-version: [10.x, 12.x]
-        os: [macOS-latest, ubuntu-latest, windows-latest]
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

```suggestion
        node-version: [10.0.x, 10.x, 12.x, 14.x, 15.x]
```
? that way you can prevent regressions